src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RankSumTest.java

```diff
@@ -114,4 +133,33 @@ protected boolean isUsableRead(final GATKRead read, final int refLoc) {
         Utils.nonNull(read);
         return read.getMappingQuality() != 0 && read.getMappingQuality() != QualityUtils.MAPPING_QUALITY_UNAVAILABLE;
     }
+
+    /**
+     * Can the base in this pileup element be used in comparative tests between ref / alt bases?
+     *
+     * Note that this function by default does not allow deletion pileup elements
```

Comment:
If we are trying to annotate a deletion site using this codepath (which we can't anyway) that it will be removed. It should also filter out any reads that correspond to a deletion at the snp site which don't have useable assignments anyway.